### PR TITLE
Monk 2524 add kubernetes status for monkrelayclusters

### PR DESCRIPTION
**Pull request by geokous**
Adding functionality to paasta to propagate the yelpsoa-configs values to the operator. Currently, this is a basic version that does not hold any information about kubernetes in the way the cassandra operator works

### CODE REVIEW

File: paasta_tools/utils.py [hunk 1]
```diff
@@ -130,6 +130,7 @@
     "flink",
     "cassandracluster",
     "kafkacluster",
+    "monkrealycluster",
```
Comment (geokous) [inline paasta_tools/utils.py:133]: @zhaoyanh1202 mispell

File: paasta_tools/monkrelaycluster_tools.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            config_dict=config_dict,
+            branch_dict=branch_dict,
+        )
+
+    def get_instances(self, with_limit: bool = True) -> int:
+        return self.config_dict.get("replicas", 1)
+
+    def validate(
+        self,
+        params: List[str] = [
```
Comment (Rob-Johnson) [inline paasta_tools/monkrelaycluster_tools.py:62]: you shouldn't use mutable values as defaults in python.

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

**Review by Rob-Johnson — APPROVED** [review]
fix and ship

File: paasta_tools/monkrelaycluster_tools.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        soa_dir=soa_dir,
+    )
+
+
+# TODO: read this from CRD in service configs
+def cr_id(service: str, instance: str) -> Mapping[str, str]:
+    return dict(
+        group="yelp.com",
+        version="v1alpha1",
+        namespace="paasta-monkrelayclusters",
```
Comment (zhaoyanh1202) [inline paasta_tools/monkrelaycluster_tools.py:139]: I don't understand this config very well, but should this be consistent with what [we defined in the yelpsoa-config](https://sourcegraph.yelpcorp.com/sysgit/yelpsoa-configs@9b66e9e7a98e5a97f9fa2049287de3c415fefff3/-/blob/monk-relay-operator/kubernetes-infrastage.yaml#L13:29)?

**Pull request merged**